### DISCUSSION TITLE
Using NSPANEL as Base Station?

### DISCUSSION THREAD

**Discussion by willyhabr** [ESPresense/ESPresense#1335] [Q&A]
Is it possible to use a NSPANEL running NSPANEL BLUEPRINT as a base station?
It is a ESP32 and is possible enable the bluetooth traking and proxy, so....

**Comment by Terastar-Paperclip** [comment]
Per our boards reference (#2334): open question, no confirmed answer yet. It's an ESP32 under the hood so it's plausible, but nobody's reported getting both the touch UI and BT scanning working over stock NSPanel firmware. If you try it, post what you find here or on #2334 and we'll fold it in.
